samples/dect/dect_phy/dect_shell/src/dect/dect_phy_ctrl.c

```diff
@@ -1108,17 +1098,41 @@ void dect_phy_ctrl_rssi_scan_stop(void)
 
 /**************************************************************************************************/
 
+static bool dect_phy_ctrl_op_ongoing(void)
+{
+	if (ctrl_data.rssi_scan_on_going) {
```

Comment:
You could also change this and several other flags below.
```suggestion
	if (ctrl_data.rssi_scan_ongoing) {
```

Reply:
Changed all